content/en/getting-started/quickstart.md

```diff
@@ -3,9 +3,103 @@ title: "Quickstart"
+{{< /alert >}}
+
+## Start your LocalStack container
+
+After installing LocalStack and configuring your API key, let us start LocalStack. You can start LocalStack via the `localstack` CLI, Docker, or a Docker Compose setup. In this guide, we will use the `localstack` CLI to start LocalStack.
+
+{{< command >}}
+$ localstack start 
+{{< / command >}}
+
+You can optionally run your LocalStack container in background mode by adding the `-d` flag to the `localstack start` command.
+
+## Setup the sample AWS application
+
+With LocalStack ready, you are ready to run your first AWS application locally. Clone our [sample application](https://github.com/localstack/localstack-demo) using Git on your local machine:
+
+{{< command >}}
+$ git clone git@github.com:localstack/localstack-demo.git
+{{< / command >}}
+
+The sample app illustrates a typical web application scenario with asynchronous request processing happening in the background, all running locally inside LocalStack.
+
+<img src="sample-app-architecture.png" alt="Application architecture for the sample application, with the different components and services involved in processing the requests." title="Application architecture for the sample application, with the different components and services involved in processing the requests." width="600" />
+
+In this sample application:
+
+- Users can send a new request through a locally deployed API Gateway which is connected to a Lambda function for further processing.
+- The Lambda function then puts the request through an SQS queue which then triggers another Lambda function that reads off the SQS queue.
+- The Lambda function then puts it on a Step Functions State Machine which consists of two steps: A processing Lambda function that processes the requests and updates a local DynamoDB table, and an archiving Lambda function that puts the result in an S3 bucket.
+- As the request is going through different stages of processing, the table below will be updated with the current request status (**QUEUED** -> **PROCESSING** -> **FINISHED**).
+- Once the request is FINISHED, the processing result will become available as a link to a file stored in an S3 bucket.
+
+All of these resources will be deployed to the local `us-east-1` region on LocalStack using Serverless framework.
+
+Let us install all the dependencies for our sample application. You need to ensure that you have Node.js and `npm` available, to install Serverless framework and other dependencies. Run the command:
+
+{{< command >}}
+$ npm install
+$ npm install -g serverless
+{{< / command >}}
+
+## Run the sample AWS application
+
+With the initial setup now completed, you can deploy the application via `serverless` CLI:
+
+{{< command >}}
+$ serverless deploy --stage local
+{{< / command >}}
+
+Let us now serve the web application locally which can be accessed through your browser:
+
+{{< command >}}
```

Comment:
nit: Given that we have pretty good support for S3 file serving (and S3 website hosting), we could skip this step, and directly upload the file to an S3 bucket and access it from there. 

Something like:
```
$ awslocal mb s3://web-app
$ awslocal s3 sync demo/web s3://web-app
```
Then we could open the file directly via `http://localhost:4566/web-app/index.html` (or `http://web-app.localhost.localstack.cloud:4566/index.html`), which would make the sample a bit more self-contained (no need to run a separate Web server to host the files).

Maybe not required for now (should also be fine to merge as-is), but I think it would be great if we can leverage our S3 for hosting Web files in the sample, going forward.. 👍

Reply:
Hi @whummer — Unfortunately, I was not able to get the Web app running by uploading it to the S3 bucket. Some GET requests were continiously failing and I was able to validate if it was due to CORS or something else.